.github/workflows/release.yml

```diff
@@ -49,6 +52,8 @@ jobs:
     if: startsWith(github.ref, 'refs/tags/v')
     runs-on: ubuntu-latest
     needs: build
+    permissions:
+      checks: write
```

Comment:
Definitely don't need `checks: write` here. Looks like this job failed last time because `grunt` wasn't installed correctly. https://github.com/amazon-ion/ion-js/actions/runs/4800737685/job/13015905967#step:7:15

Reply:
I am not sure about why the `grunt` command shows not found. In the build script it seems to work fine.

Reply:
I think we should probably add `run: npm ci` before doing `run: npm publish` to solve installing grunt or any other dependencies which might be needed for publish.  
From the [github documents](https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages) on publishing node.js package:
```
 - run: npm ci
 - run: npm publish
        env:
          NODE_AUTH_TOKEN: ${{ GITHUB_TOKEN }}
```